src/app/publications/publications.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="mat-typography">
+  <div *ngIf="errorMessage" class="publications-error-message">{{errorMessage}}</div>
+  <div class="publications-toolbar">
+    <button mat-raised-button (click)="openNewDialog()" aria-label="Add testimonial">Add publication</button>
```

Comment:
Please change `aria-label` from `Add testimonial` to `Add publication`.

Reply:
thank you. done